Copies Atlas CLI commands for release tag: atlascli/v1.34.0 (#803)
mongodb/docs-atlas-cli: source/command/atlas-deployments-setup.txt

     - 
     - false
     - help for setup
   * - --initdb
     - string
     - false

>      - Flag that uses a folder to be mapped into LOCAL deployment for initialization

Reviewer: ```suggestion
     - Flag to use a folder to map into LOCAL deployment for initialization.
```
